sdk/cosmosdb/cosmos/test/internal/unit/defaultQueryExecutionContext.spec.ts

```diff
@@ -7,6 +7,8 @@ import {
 import { FeedOptions } from "../../../src";
 import assert from "assert";
 import { sleep } from "../../../src/common";
+import { getEmptyCosmosDiagnostics } from "../../../src/CosmosDiagnostics";
+import { CosmosDiagnosticContext } from "../../../src/CosmosDiagnosticsContext";
```

Comment:
Use the already imported line getEmptyCosmosDiagnostics ?

Reply:
Hey didn't understand this line, please explain. getEmptyCosmosDiagnostics is imported only once.